batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/constants/AppConstant.java

```diff
@@ -0,0 +1,11 @@
+package com.lftechnology.batch7crud.constants;
+
+/**
+ * Created by sanjay on 1/22/16.
+ */
+public class AppConstant {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)